Marsiple/Common/Models/Comment.swift

```diff
@@ -22,4 +22,29 @@ class Comment: Codable {
         self.body = body
         self.postId = postId
     }
+    
+    private enum CodingKeys: String, CodingKey {
+        case id
+        case name
+        case email
+        case body
+        case postId
+    }
+    
+    required init(from decoder: Decoder) throws {
```

Comment:
We don't need the `decoder` specified. By `default`, iOS will use all properties within a `Codable` `type` for `decoding`.